File: darktenary.py
#Dark Tenary Test - Python
def getDayOfWeek(dayNumber):
    return "Sunday" if dayNumber == 0 else "Monday" if dayNumber == 1 else "Tuesday" if dayNumber == 2 else "Wednesday" if dayNumber == 3 else "Thursday" if dayNumber == 4 else "Friday" if dayNumber == 5 else "Saturday" if dayNumber == 6 else "Invalid Day Number"

print(getDayOfWeek(3))
print(getDayOfWeek(10)) 

File: darktenary.java
public class darktenary {
    public static String getDayOfWeek(int dayNumber) {
        return dayNumber == 0 
        ? "Sunday"
        : dayNumber == 1
        ? "Monday"
        : dayNumber == 2
        ? "Tuesday"
        : dayNumber == 3
        ? "Wednesday"
        : dayNumber == 4
        ? "Thursday"
        : dayNumber == 5
        ? "Friday"
        : dayNumber == 6
        ? "Saturday"
        : "Invalid Day Number";
    }

    public static void main(String[] args) {
        System.out.println(getDayOfWeek(3));
        System.out.println(getDayOfWeek(10));
    }
}